src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -972,8 +954,8 @@ public Long getLong() {
             if (data instanceof Long longValue) {
```

Comment:
Left this as this is exactly the return type. Ok?

Reply:
For simplicity sake I would include it in the Number case and remove this if. When looking at the Implementation of the Integer or Long class they do exactly the same